.gitlab-ci.yml

```diff
@@ -21,6 +21,23 @@ generate-scripts:
       - install_script_agent6.sh
       - install_script_agent7.sh
 
+test-opensuse-13:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:docker"]
+  stage: test
+  dependencies: ["generate-scripts"]
+  parallel:
+    matrix:
+      - IMAGE: images/mirror/opensuse/archive:13.2
+      - IMAGE: images/mirror/opensuse/archive:13.2
+        FLAVOR: datadog-dogstatsd
+      - IMAGE: images/mirror/opensuse/archive:13.2
+        FLAVOR: datadog-iot-agent
+  script:
+    - ./test/dockertest.sh --image "registry.ddbuild.io/${IMAGE}" --script install_script_agent6.sh --minor-version "${MINOR_VERSION}" --expected-minor-version "${EXPECTED_MINOR_VERSION}" --flavor "${FLAVOR}" --suse-old "true"
+    - ./test/dockertest.sh --image "registry.ddbuild.io/${IMAGE}" --script install_script_agent7.sh --minor-version "${MINOR_VERSION}" --expected-minor-version "${EXPECTED_MINOR_VERSION}" --flavor "${FLAVOR}" --suse-old "true"
```

Comment:
you could add the agent version as part of the matrix, wdyt?
```
parallel:
    matrix:
      - IMAGE: opensuse/archive:13.2
        FLAVOR: ['', datadog-dogstatsd, datadog-iot-agent]
        VERSION: [agent6, agent7]
script:
    - ./test/dockertest.sh --image "registry.ddbuild.io/images/mirror/${IMAGE}" --script install_script_"${VERSION}".sh --minor-version "${MINOR_VERSION}" --expected-minor-version "${EXPECTED_MINOR_VERSION}" --flavor "${FLAVOR}" --suse-old "true"
```
Not sure the empty flavour would work, you might have to keep the first line as is

Reply:
I could but since the install_script is not a big load for the runner I figured it would make more sense to not instantiate a a whole new instance only for a small difference. And I think since it's the way it's done in the script previously we could stay consistent with how it's done or edit all the other jobs as well but I think it's fine as it is. wdyt ?

Reply:
ok to keep like this, you are right it may save some compute (you can maybe just group the flavours and have the same behaviour as today)